ui/tsconfig.json

```diff
@@ -1,26 +1,28 @@
 {
   "compilerOptions": {
     "target": "es5",
-    "lib": [
-      "dom",
-      "dom.iterable",
-      "esnext"
-    ],
+    "lib": ["dom", "dom.iterable", "esnext"],
+    "paths": {
```

Comment:
seems like this will have ramifications on all devs @Midnite1212 @yuenyauu  @samyakobus be aware